### PR TITLE
EOL Analytics, Chef Compliance, Enterprise Chef, and Reporting

### PR DESCRIPTION
**Pull request by kagarmoe**
### Description

Removes Analytics, Chef Compliance, Enterprise Chef, and Reporting pages from chef-web-docs. Removes references to these products from the remaining pages, where possible. Changes "Chef Open Source" to "Chef". Applies traditional copyright.

### Check List

- [x] Spell Check
- [x] Local build
- [x] Examine the local build
- [ ] All tests pass

### CODE REVIEW

File: chef_master/source/platform_overview.rst [hunk 1]
```diff
@@ -115,18 +115,10 @@ Compliance
 
 Chef Automate creates customizable reports that identify compliance issues, security risks, and outdated software. You can write your own compliance rules in InSpec, or you can get started quickly by using built-in profiles, which are predefined rule sets for a variety of security frameworks, such as Center for Internet Security (CIS) benchmarks, included as part of Chef Automate.
 
-Compliance reporting is currently done through two mechanisms: The integrated compliance reporting in Chef Automate 0.8.5 or later, and the standalone Chef Compliance server.
+Compliance reporting is currently done through the integrated compliance reporting in Chef Automate 0.8.5.
```
Comment (teknofire) [inline chef_master/source/platform_overview.rst]: Do we still want to include the version number here?

Comment (kagarmoe) [reply chef_master/source/platform_overview.rst]: Nope. Removed them from everywhere except for "Upgrade" and the release notes"

**Review by mjingle — APPROVED** [review]
tenor-260653704

**Comment by kagarmoe** [comment]
tenor-46758461

**Pull request merged**